### PR TITLE
Wrangle some python deps

### PR DESCRIPTION
**Pull request by vchiapaikeo**
Python 3.6 loses support in github actions: https://github.com/actions/setup-python/issues/544

Dropping it as a status check here. Will do a minor/major bump later to formally remove support for it in setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,13 +43,14 @@
...
         'jsonschema>=2.6.0,<3.0',
-        'jinja2>=2.8.1,<3.0',
+        'jinja2>=2.8.1',
         'pyyaml>=4.2b1',
         'marshmallow>=2.13.6,<3.0',
         'networkx>=2.4,<2.5',
         'xmltodict>=0.11.0,<1.0',
         'six>=1.11.0,<2.0',
         'markupsafe==2.0.1',
+        'importlib-metadata<5.0',
```
Comment (li-roy) [inline setup.py:53]: What was this needed for?

**Review by li-roy — APPROVED** [review]
Thanks for the quick follow up!

File: setup.py (hunk 1 above)
Comment (vchiapaikeo) [reply setup.py:53]: Test failures in newer versions of importlib-metadata. I can perform this release tmrw morning.

**Comment by vchiapaikeo** [comment]
1.9.18 is now available: https://pypi.org/project/boundary-layer/#history

**Pull request merged**